parlai/core/params.py

```diff
@@ -788,8 +798,3 @@ def ag_add_argument(*args, **kwargs):
 
         arg_group.add_argument = ag_add_argument  # override _ => -
         return arg_group
-
-    def error(self, message):
```

Comment:
is there a reason we are removing this?

Reply:
it looked unused to me. DEPRECATE ALL THE THINGS

Reply:
Actually, this specific function is related to the "Arg failures errors should happen AFTER the help message, not before" doc day task. It is what is called when there is an argparse error with our parser (you can un-delete it and i will take care of it tomorrow)